master/internal/core.go

```diff
@@ -976,6 +976,11 @@ func (m *Master) Run(ctx context.Context, gRPCLogInitDone chan struct{}) error {
 		return errors.Wrap(err, "could not fetch cluster id from database")
 	}
 
+	err = user.UpdateDefaultUserPasswords(ctx, m.config.Security)
```

Comment:
I thought that's only supposed to trigger on new database creation, no?

Reply:
I was going with a scheme like: if they are supplied, update them, if they are not supplied, don't change them.